tests/core/a-assets.test.js

```diff
@@ -246,12 +265,40 @@ suite('a-asset-item', function () {
     assetItem.setAttribute('src', 'doesntexist');
     assetItem.addEventListener('error', function (evt) {
       assert.ok(evt.detail.xhr !== undefined);
+      evt.stopPropagation();
```

Comment:
OMG this line! This is a good find @diarmidmackenzie 
Without this line, it reduced silently the number of tests that was run from 1121 to 559 while I was working on #5123